### PR TITLE
Check for flagged status when adding a new source

### PR DESCRIPTION
**Pull request by Skittyblock**

### CODE REVIEW

File: Sileo/UI/SourcesViewController/FlaggedSourceWarningViewController/FlaggedSourceWarningViewController.swift [hunk 1]
```diff
@@ -22,6 +22,8 @@ class FlaggedSourceWarningViewController: UIViewController {
     
     @objc var shouldAddAnywayCallback: (() -> Void)?
     
+    var url: URL?
```
Comment (coolstar) [inline Sileo/UI/SourcesViewController/FlaggedSourceWarningViewController/FlaggedSourceWarningViewController.swift:25]: looks like this variable is not used

**Comment by nullpixel** [comment]
I’ll have a look at this later today :)

**Comment by sinekrs** [comment]
Xxxx

**Comment by sinekrs** [comment]
Xxxx

**Comment by sinekrs** [comment]
Xxxx

**Pull request merged**